src/info/langs/mod.rs

```diff
@@ -66,146 +73,13 @@ fn get_statistics(
     globs_to_exclude: &[String],
     language_types: &[LanguageType],
     include_hidden: bool,
-) -> tokei::Languages {
-    let mut languages = tokei::Languages::new();
-    let filtered_languages = filter_languages_on_type(language_types);
-
-    let tokei_config = tokei::Config {
-        types: Some(filtered_languages),
-        hidden: Some(include_hidden),
-        ..tokei::Config::default()
-    };
-    let ignored: Vec<&str> = globs_to_exclude.iter().map(AsRef::as_ref).collect();
-    languages.get_statistics(&[&dir], &ignored, &tokei_config);
-    languages
+) -> Result<gengo::Analysis, Box<dyn Error>> {
+    // TODO Determine best way to ignore files (and if that should continue to be handled by onefetch)
+    let file_source = Git::new(dir, "HEAD")?;
```

Comment:
is this going to take into account the pending changes ?

Reply:
https://github.com/o2sh/onefetch/discussions/1303#discussioncomment-9057210